diesel/src/expression/sql_literal.rs

```diff
@@ -139,6 +139,12 @@ impl<ST> NonAggregate for SqlLiteral<ST> {
 /// DSL. You will need to provide the SQL type of the expression, in addition to
 /// the SQL.
 ///
+/// # Bound parameters
+///
+/// If you need to pass arguments to your query, you should use [`.bind()`].
+///
+/// [`.bind()`]: struct.SqlLiteral.html#method.bind
```

Comment:
No it doesn't... I think I didn't wait for `cargo doc` to finish before testing.